pkg/render/apiserver.go

```diff
@@ -290,6 +294,25 @@ func (c *apiServerComponent) Ready() bool {
 	return true
 }
 
+func (c *apiServerComponent) apiServerPodDisruptionBudget() *policyv1.PodDisruptionBudget {
+	maxUnavailable := intstr.FromInt(1)
+	return &policyv1.PodDisruptionBudget{
+		TypeMeta: metav1.TypeMeta{Kind: "PodDisruptionBudget", APIVersion: "policy/v1"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      common.APIServerDeploymentName,
+			Namespace: common.CalicoNamespace,
+		},
+		Spec: policyv1.PodDisruptionBudgetSpec{
+			MaxUnavailable: &maxUnavailable,
+			Selector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{
+					AppLabelName: APIServerK8sAppName,
```

Comment:
I think this needs to match the labels set on the deployment selector here: https://github.com/tigera/operator/blob/4eff4995da85134f91a4a7c3bebf072b1095f35e/pkg/render/apiserver.go#L997

Maybe we should commonize this as well into a new private function in this file - `deploymentSelector` - so that both places can use it and make sure they have the same values.

Reply:
Please review changes @caseydavenport